packages/ppx/test/native/Static_test.re

```diff
@@ -311,6 +311,25 @@ let properties_static_css_tests = [
     [%expr [%css "background-color: red"]],
     [%expr CssJs.backgroundColor(CssJs.red)],
   ),
+  (
+    [%css "background-color: color-mix(in srgb, white)"],
+    [%expr [%css "background-color: color-mix(in srgb, white)"]],
```

Comment:
Still wondering why this errors 
```Error: Property 'background-color' has an invalid value: 'color-mix(in srgb, white)'```

@davesnx, what do you think?

Reply:
It seems like the parsing logic isn't working. Probably the description is wrong (Parser.re)

Reply:
Indeed, I missed quoting a comma. haha!

Reply:
Thanks